rust/release-checklist.md

```diff
@@ -38,19 +38,30 @@ Push access to the upstream repository is required in order to publish the new t
 
 :warning:: if `origin` is not the name of the locally configured remote that points to the upstream git repository (i.e. `git@github.com:coreos/{{ git_repo }}.git`), be sure to assign the correct remote name to the `UPSTREAM_REMOTE` variable.
 
+- declare environment variables
+  - [ ] `RELEASE_VER=x.y.z`
+  - [ ] `UPSTREAM_REMOTE=origin`
+
+- update all dependencies
+  - [ ] `git checkout -b pre-release-${RELEASE_VER}`
+  - [ ] `cargo update`
+  - [ ] `git add Cargo.lock && git commit -m "cargo: update dependencies"`
```

Comment:
I forgot that `Cargo.lock` only gets committed to the repo for applications and not for libraries.  So... we'll need to conditionalize this entire section.

Reply:
Ah great point.